packer/instance-scripts/init-bootnode.sh

```diff
@@ -79,15 +72,18 @@ wait_for_successful_command 'vault status'
 # Wait for vault to be fully configured by the root user
 wait_for_successful_command 'vault auth -method=aws'
 
-wait_for_terraform_provisioners
-
 # Get metadata for this instance
 INDEX=$(cat /opt/quorum/info/index.txt)
 AWS_REGION=$(cat /opt/quorum/info/aws-region.txt)
-PUBLIC_IP=$(wait_for_successful_command 'curl http://169.254.169.254/latest/meta-data/public-ipv4')
+PUBLIC_IP=$(cat /opt/quorum/info/public_ip.txt)
+EIP_ID=$(cat /opt/quorum/info/eip_id.txt)
+INSTANCE_ID=$(wait_for_successful_command 'curl -s http://169.254.169.254/latest/meta-data/instance-id')
 HOSTNAME=$(wait_for_successful_command 'curl http://169.254.169.254/latest/meta-data/public-hostname')
 BOOT_PORT=30301
 
+# Associate the EIP with this instance
```

Comment:
Yup!  This only needs to happen when the node turns on, then the EIP is associated until it's explicitly reclaimed or reassociated with a new node.  When a replacement node spins up and re-runs the command, it will then get this IP.